samples/openthread/ncp/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <zephyr.h>
+#include <logging/log.h>
+#include "openthread-system.h"
```

Comment:
Why is this needed?

Reply:
Probably I left it after testing something. Done